### PR TITLE
(Examples) update Grafbase example

### PR DESCRIPTION
**Pull request by vinaykulk621**

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: fa13b37c7fe900148970ea53475aad9a7f4b5dc2

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: examples/with-grafbase/app/layout.tsx [hunk 1]
```diff
@@ -4,12 +4,14 @@ import Link from 'next/link'
 
 import { graphql } from '../gql'
 import { grafbase } from '../lib/grafbase'
+import type { Metadata } from "next";
```
Comment (balazsorban44) [inline examples/with-grafbase/app/layout.tsx:7]:
```suggestion
import type { Metadata } from "next"
```

File: examples/with-grafbase/app/layout.tsx [hunk 2]
```diff
@@ -4,12 +4,14 @@ import Link from 'next/link'
...
 
 export const revalidate = 0
 
-export const metadata = {
+export const metadata: Metadata = {
   title: 'Grafbase + Next.js',
-}
+  description: 'Grafbase + Next.js',
+};
```
Comment (balazsorban44) [inline examples/with-grafbase/app/layout.tsx:14]:
```suggestion
}
```

File: examples/with-grafbase/app/layout.tsx [hunk 3]
```diff
@@ -80,7 +80,5 @@ const RootLayout = async ({ children }: { children: React.ReactNode }) => {
         </div>
       </body>
     </html>
-  )
+  );
```
Comment (balazsorban44) [inline examples/with-grafbase/app/layout.tsx:83]:
```suggestion
  )
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thanks, remember to run `pnpm lint` when updating an example

**Comment by vinaykulk621** [comment]
@balazsorban44  Any updates on this??

**Review by balazsorban44 — DISMISSED** [review]

**Comment by balazsorban44** [comment]
Thanks, note that you don't need to keep merging `canary`, it won't result in a faster merge. :+1:

**Review by leerob — DISMISSED** [review]

**Comment by leerob** [comment]
Linting is failing here, FYI

**Comment by vinaykulk621** [comment]
@leerob Fixed linting errors

**Review by leerob — APPROVED** [review]

**Review by leerob — DISMISSED** [review]

**Comment by vinaykulk621** [comment]
@leerob this time i fixed it believe me, please!!

**Review by leerob — DISMISSED** [review]

**Review by leerob — DISMISSED** [review]

**Comment by leerob** [comment]
Did you fix the Prettier errors? It's still failing. Are you validating locally? 🙏

**Comment by vinaykulk621** [comment]
I am validating in github codespace here is the video on how i am trying to do


Nothing seems to change here tho 

https://github.com/vercel/next.js/assets/94120295/52de6341-8d71-44fe-a2e8-9ff327751147




i tried running `pnpm lint` at the root directory but it's not working either 


image

I tried to changing the `pnpm` version but it does not seem to work here.
It goes back to `8.6.1` even though i run `pnpm i -g pnpm@8.7.1`

 
i thought i should run it in the example directory where i am making changes but the example do not include lint script in `package.json`. 

You could say i really have no idea where i should run `pnpm lint`.
Here in [linting.md](https://github.com/vercel/next.js/blob/canary/contributing/repository/linting.md#linting) it is not specfied where i should i run this command when i update an examples.

Please help🙏🏻.

**Comment by manovotny** [comment]
@vinaykulk621 run `corepack enable pnpm` to get the pnpm versions in sync. Then, run `pnpm prettier-fix` from the root of the project and commit the changes.

**Review by leerob — APPROVED** [review]

**Pull request merged**